client/my-sites/media-library/data-source.jsx

```diff
@@ -62,6 +63,11 @@ export class MediaLibraryDataSource extends Component {
 				label: translate( 'Pexels free photos' ),
 				icon: <PexelsIcon className="gridicon" />, // eslint-disable-line wpcalypso/jsx-classname-namespace
 			} );
+			sources.push( {
+				value: 'openverse',
+				label: translate( 'Openverse free photos' ),
+				icon: <OpenverseIcon className="gridicon" />, // eslint-disable-line wpcalypso/jsx-classname-namespace
+			} );
```

Comment:
Can this please be put behind a feature flag? If the Openverse API can't take the new load from Calypso and needs to be scaled, it would be nice to prevent requests from coming in during that time (and also prevent a confusing and disappointing user experience).

Reply:
Good call. Updated in e6e1996151e865107b1cb1d31ddd8b2e1e90bcbb